test/otel/testdata/builder-config.yaml

```diff
@@ -1,10 +1,11 @@
-connectors:
-- gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector
-    v0.114.0
 dist:
   description: Basic OTel Collector distribution for Developers
   name: otelcol-custom
   output_path: /tmp/otel-ci/otelcol-custom
+connectors:
+- gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector
+    v0.114.0
+- gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector v0.114.0
```

Comment:
Was this missing previously? Is it a new req?

Reply:
wanted to make sure converter works with the connector properly:
https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/converter/README.md?plain=1#L31-L33